exercises/02-publish-subscribe/after/Divergent.Customers/Handlers/OrderSubmittedHandler.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using System.Data.Entity;
 using System.Threading.Tasks;
-using Divergent.Customers.Data.Context;
-using Divergent.Sales.Messages.Events;
 using NServiceBus;
 using NServiceBus.Logging;
+using Divergent.Customers.Data.Context;
+using Divergent.Sales.Messages.Events;
```

Comment:
This is an unrelated change. It should really be in a separate commit or PR. Same in a few other places.